projects/msvc/GLideN64.vcxproj

```diff
@@ -46,7 +46,7 @@
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
     <CodeAnalysisRules />
     <CodeAnalysisRuleAssemblies />
-    <OutDir>$(SolutionDir)..\..\..\external\$(Platform)\</OutDir>
+    <OutDir>$(MSBuildProjectDirectory)\..\..\..\external\$(Platform)\</OutDir>
     <IntDir>$(SolutionDir)build\$(Platform)\$(Configuration)\$(ProjectName)\</IntDir>
```

Comment:
None of the project file changes appear relevant to the PR, and should be removed.